### PR TITLE
DAOS-2698: Fix Unittest code.

### PR DESCRIPTION
**Pull request by ravalsam**
- test need /mnt/daos mounted, Jenkins CI pipeline is not mounting
      /mnt/daos on local machine from where Avocado test initiated.
    - Timeout is needed now as vos_tests is taking ~4 min.

Test-tag:unittest

Signed-off-by: Samir <samir.raval@intel.com>

### CODE REVIEW

File: src/tests/ftest/unittest/Unittest.py [hunk 1]
```diff
@@ -45,9 +45,10 @@ def unittest_runner(self, unit_testname):
         """
         name = self.params.get("testname", '/run/UnitTest/{0}/'
                                .format(unit_testname))
+        server = self.params.get("test_machines", "/run/hosts/*")
         bin_path = get_file_path(name, "install/bin")
 
-        cmd = ("{0}".format(bin_path[0]))
+        cmd = ("ssh {} {}".format(server[0], bin_path[0]))
```
Comment (brianjmurrell) [inline src/tests/ftest/unittest/Unittest.py]: Why not use the new ClusterShell support being added?

File: src/tests/ftest/unittest/Unittest.py [hunk 2]
```diff
@@ -56,34 +57,36 @@ def unittest_runner(self, unit_testname):
...
 
     def test_vea_ut(self):
         """
         Test vea unittest.
-        :avocado: tags=unittest,nvme,vea_ut
+        :avocado: tags=all,unittest,pr,tiny,hw,vea_ut
         """
         self.unittest_runner("vea_ut")
 
+    @skipForTicket("DAOS-1766")
```
Comment (brianjmurrell) [inline src/tests/ftest/unittest/Unittest.py]: This ticket is Resolved.

**Comment by ravalsam** [comment]
This is not ready for review.

File: src/tests/ftest/unittest/Unittest.py (hunk 1 above)
Comment (ravalsam) [reply src/tests/ftest/unittest/Unittest.py]: yes was thinking of doing it but want to see if test-tag is picking and it's working in Ci..

**Comment by brianjmurrell** [comment]
> This is not ready for review.

That's fair enough.  I was just trying to provide early input to reduced the cycles later on.  And since you didn't solicit my review please feel free to force push if you want.  IMO, it's only once you have started to get **solicited** reviews that you cannot force-push any more.

File: src/tests/ftest/unittest/Unittest.py (hunk 1 above)
Comment (brianjmurrell) [reply src/tests/ftest/unittest/Unittest.py]: Seems it is:
```
01:24:30.725  Detected tests:  
01:24:30.725  ./daos_test/daos_core_test.py  
01:24:30.725  ./unittest/Unittest.py
```

and https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-641/2/testReport/(root)/UnitTest/

**Comment by ravalsam** [comment]
> > This is not ready for review.
> 
> That's fair enough. I was just trying to provide early input to reduced the cycles later on. And since you didn't solicit my review please feel free to force push if you want. IMO, it's only once you have started to get **solicited** reviews that you cannot force-push any more.

Well it's nothing bad to review early but I am waiting for those two tickets to be fixed so this can be fixed with all unit tests pass. 
Other reason is Saurabh has one more PR https://github.com/daos-stack/daos/pull/606 which I need to make sure gets debased on whichever gets landed first. He has the +2 on review so it may land soon.
For Clustershell,  I am not sure when PR#400 will get merge so I need to copy the https://github.com/daos-stack/daos/pull/400/files#diff-934811324dbd9210dcacc7500f41d318R310 in current PR.

**Comment by ravalsam** [comment]
This PR is ready for review, Clustershell improvement has been removed. clustershell will be included as part of separate improvements ticket [https://jira.hpdd.intel.com/browse/DAOS-2750](https://jira.hpdd.intel.com/browse/DAOS-2750)

**Pull request merged**